webpack.config.js

```diff
@@ -56,7 +56,10 @@ module.exports = {
       test: /\.js$/,
       exclude: /node_modules/,
       use: [{
-        loader: 'babel-loader'
+        loader: 'babel-loader',
+        options: {
+          presets: [['@babel/preset-env', { modules: false }]]
```

Comment:
For future understanding of `{ modules: false }`:
https://stackoverflow.com/a/47675519